content/departments/engineering/teams/search/guiding_principles.md

```diff
@@ -1,41 +0,0 @@
-# Guiding principles for teamwork on Search Product
```

Comment:
It would be great to develop something similar for our team. I personally really liked these guiding principles, and I think they would make a great starting point.